lib/avro/builder/dsl.rb

```diff
@@ -89,6 +89,25 @@ def type(*)
         @last_object = super
       end
 
+      # Several variants:
+      # define_type(:foo, long)
+      # define_type(:bar, { namespace: 'com.example' }, long)
+      # define_type(:baz, 'com.example', long)
```

Comment:
Will rework to make the signature `type_alias(name, type_object, options = {})`.